src/main/java/com/cburch/logisim/std/ttl/Ttl7400.java

```diff
@@ -28,9 +28,8 @@
 
   private static class NandGateHDLGeneratorFactory extends AbstractGateHDLGenerator {
 
-    @Override
-    public String getComponentStringIdentifier() {
-      return "TTL7400";
+    public NandGateHDLGeneratorFactory() {
+      super("TTL7400");
```

Comment:
Would that make sense to rework this code a bit and instead of using `TTL7400` string, prefix `_ID` with `TTL` to avoid duplication (multiple places)?